From 7b0ba61bfbe7765d034f82158eba57c74b770405 Mon Sep 17 00:00:00 2001
From: Karolina Surma <ksurma@redhat.com>
Date: Mon, 21 Feb 2022 09:22:42 +0100
Subject: [PATCH] Update

---
 setup.cfg | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 8c71052..e6df94f 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -38,19 +38,8 @@ exclude =
 [options.extras_require]
 testing = 
 	pytest >= 6
-	pytest-checkdocs >= 2.4
-	pytest-flake8
-	pytest-black >= 0.3.7; \
-	python_implementation != "PyPy"
-	pytest-cov; \
-	python_implementation != "PyPy"
-	pytest-mypy >= 0.9.1; \
-	python_implementation != "PyPy"
-	pytest-enabler >= 1.0.1
-	pytest-perf
 	
 	mock
-	flake8-2020
 	virtualenv>=13.0.0
 	wheel
 	pip>=19.1 # For proper file:// URLs support.
@@ -58,8 +47,6 @@ testing =
 	pytest-xdist
 	jaraco.path>=3.2.0
 	build[virtualenv]
-	filelock>=3.4.0
-	pip_run>=8.8
 	ini2toml[lite]>=0.9
 	tomli-w>=1.0.0
 testing-integration = 
-- 
2.35.3

