From bc29357f2effabd91c47562d332f0ae731ffcee0 Mon Sep 17 00:00:00 2001
From: Karolina Surma <ksurma@redhat.com>
Date: Mon, 21 Feb 2022 09:22:42 +0100
Subject: [PATCH] Update

---
 setup.cfg | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 4a4a9a2..ce587f6 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -38,19 +38,8 @@ exclude =
 [options.extras_require]
 testing = 
 	pytest >= 6
-	pytest-checkdocs >= 2.4
-	pytest-flake8
-	pytest-black >= 0.3.7; \
-	python_implementation != "PyPy"
-	pytest-cov; \
-	python_implementation != "PyPy"
-	pytest-mypy >= 0.9.1; \
-	python_implementation != "PyPy"
-	pytest-enabler >= 1.0.1
-	pytest-perf
 	
 	mock
-	flake8-2020
 	virtualenv>=13.0.0
 	wheel
 	pip>=19.1 # For proper file:// URLs support.
@@ -58,9 +47,6 @@ testing =
 	pytest-xdist
 	sphinx>=4.3.2
 	jaraco.path>=3.2.0
-	build[virtualenv]
-	filelock>=3.4.0
-	pip_run>=8.8
 testing-integration = 
 	pytest
 	pytest-xdist
-- 
2.35.1

